### PR TITLE
Revert "workaround: Uninstall btrfsmaintenance bsc#1174436"

### PR DESCRIPTION
**Pull request by dzedro**
This reverts commit 5ba4f9c26f03ba594f87d93244c432c17d8d5219.

- Related ticket: https://bugzilla.suse.com/show_bug.cgi?id=1174436
- Verification run: https://openqa.suse.de/tests/5505655#step/autoyast_reboot/1

### CODE REVIEW

File: tests/autoyast/autoyast_reboot.pm [hunk 1]
```diff
@@ -25,13 +25,8 @@ use base 'basetest';
 use testapi;
 use power_action_utils 'power_action';
 use version_utils 'is_sle';
```
Comment (b10n1k) [inline tests/autoyast/autoyast_reboot.pm:27]: this should also be removed IMO.

**Review by b10n1k — CHANGES_REQUESTED** [review]
i guess that `version_utils` was loaded here even before the reverted commit but it looks unused. Could you remove it?

File: tests/autoyast/autoyast_reboot.pm (hunk 1 above)
Comment (dzedro) [reply tests/autoyast/autoyast_reboot.pm]: Thanks, removed

**Review by b10n1k — APPROVED** [review]
LGTM

**Pull request merged**